apps/meteor/app/cloud/server/functions/getWorkspaceLicense.ts

```diff
@@ -44,14 +41,19 @@ const fetchCloudWorkspaceLicensePayload = async ({ token }: { token: string }):
 
 	const payload = await response.json();
 
-	assertWorkspaceLicensePayload(payload);
+	const assertWorkspaceLicensePayload = workspaceLicensePayloadSchema.safeParse(payload);
+
+	if (!assertWorkspaceLicensePayload.success) {
+		console.error('workspaceLicensePayloadSchema failed type validation', assertWorkspaceLicensePayload.error.errors);
```

Comment:
please avoid using `console` as much as possible, use an instance of `Logger` instead. in this particular file there is already an import for `SystemLogger` which is a generic instance of `Logger`.

```suggestion
		SystemLogger.error({ msg: 'workspaceLicensePayloadSchema failed type validation', errors: assertWorkspaceLicensePayload.error.errors });
```